services/galley/src/Galley/API/MLS/Util.hs

```diff
@@ -123,3 +90,30 @@ getPendingBackendRemoveProposals gid epoch = do
             TinyLog.warn $ Log.msg ("found pending proposal without origin, ignoring" :: ByteString)
             pure Nothing
       )
+
+gtcToConv ::
+  GlobalTeamConversation ->
+  UserId ->
+  [LocalMember] ->
+  Conversation
+gtcToConv gtc usr lm =
+  let mlsData = gtcMlsMetadata gtc
+   in Conversation
+        { convId = qUnqualified $ gtcId gtc,
+          convLocalMembers = lm,
+          convRemoteMembers = mempty,
+          convDeleted = False,
+          convMetadata =
+            ConversationMetadata
+              { cnvmType = GlobalTeamConv,
+                -- FUTUREWORK: Make this a qualified user ID.
```

Comment:
That FUTUREWORK is proably better placed at the definition of `ConversationMetadata`

Reply:
It is already there. Copy paste remnants :)